File: create_test_user.dart
import 'lib/common/database_helper.dart';

void main() async {
  print('🔧 Creating test user for Nine27 Pharmacy...\n');
  
  try {
    final dbHelper = DatabaseHelper();
    
    // Create a test user
    final result = await dbHelper.registerUser(
      name: 'Test User',
      email: 'test@nine27pharmacy.com',
      mobile: '09123456789',
      address: '123 Test Street, Test City',
      password: 'password123',
    );
    
    if (result['success']) {
      print('✅ Test user created successfully!');
      print('📧 Email: test@nine27pharmacy.com');
      print('🔑 Password: password123');
      print('\n🎯 You can now use these credentials to login in your app.');
    } else {
      print('❌ Failed to create test user: ${result['message']}');
    }
  } catch (e) {
    print('❌ Error: $e');
  }
}

File: nine27-pharmacy-backend/create_test_user.php
<?php

require_once 'vendor/autoload.php';

use Illuminate\Support\Facades\Hash;
use App\Models\User;

// Bootstrap Laravel
$app = require_once 'bootstrap/app.php';
$app->make('Illuminate\Contracts\Console\Kernel')->bootstrap();

try {
    // Check if user already exists
    $existingUser = User::where('email', 'test@example.com')->first();
    
    if ($existingUser) {
        echo "Test user already exists with email: test@example.com\n";
        echo "Password: password123\n";
        exit;
    }

    // Create test user
    $user = User::create([
        'name' => 'Test User',
        'email' => 'test@example.com',
        'mobile' => '1234567890',
        'address' => '123 Test Street, Test City',
        'password' => Hash::make('password123'),
    ]);

    echo "Test user created successfully!\n";
    echo "Email: test@example.com\n";
    echo "Password: password123\n";
    echo "User ID: " . $user->id . "\n";

} catch (Exception $e) {
    echo "Error creating test user: " . $e->getMessage() . "\n";
}





